tests/cross/src/test/scala/tests/pc/CompletionCaseSuite.scala

```diff
@@ -248,9 +248,9 @@ class CompletionCaseSuite extends BaseCompletionSuite {
        |case Some(value) => scala
        |""".stripMargin,
   )
-
+  // for scala 3.2.1 we see subclasses like `Tuple2$mcCC$sp`. Might be an issue in Dotty
```

Comment:
Can you open an issue in dotty and link to the issue from the comment?

Reply:
It was easier to fix on our side, by not going through subclasses and scope symbols for tuple types